css/components/badges.css

```diff
@@ -1,99 +1,99 @@
 .badge-wrapper {
-	min-width: 2rem;
-	padding: 1rem;
-	font-size: 1.5rem;
+  min-width: 2rem;
+  padding: 1rem;
+  font-size: 1.5rem;
 }
 
 .badge {
-	position: relative;
+  position: relative;
 }
 
 .badge-counter {
-	position: absolute;
-	font-size: 0.8rem;
-	min-width: 1.5rem;
-	min-height: 1.5rem;
-	display: flex;
-	align-items: center;
-	justify-content: center;
+  position: absolute;
+  font-size: 0.8rem;
+  min-width: 1.5rem;
+  min-height: 1.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
 .badge-top-right {
-	top: 0.25rem;
-	right: 0.25rem;
+  top: 0.25rem;
+  right: 0.25rem;
 }
 
 .badge-top-left {
-	top: 0.25rem;
-	left: 0.25rem;
+  top: 0.25rem;
+  left: 0.25rem;
```

Comment:
I think these two could be clubbed together

Reply:
One is top-right and another one is top-left

Reply:
Oooo sorry my bad😅